### ISSUE TITLE
close sidebar when click any blank part

### ISSUE THREAD

**Issue by roastduckcd** [theme-next/hexo-theme-next#606]
### I agree and want to create new issue 

- [x] Yes, I was on [Hexo Docs page](https://hexo.io/docs/), especially on [Templates](https://hexo.io/docs/templates.html), [Variables](https://hexo.io/docs/variables.html), [Helpers](https://hexo.io/docs/helpers.html) and [Troubleshooting](https://hexo.io/docs/troubleshooting.html).
- [x] Yes, I was on [NexT Documentation Site](http://theme-next.org/docs/).
- [x] And yes, I already searched for current [issues](https://github.com/theme-next/hexo-theme-next/issues?utf8=%E2%9C%93&q=is%3Aissue) and this is not help to me.

***

### Expected behavior 
can it possible to add a new feature about closing sidebar while click blank area? 

### Actual behavior 
right now, it can only be done by clicking the "sidebar-toggle" button at the lower right.

#### Hexo version

```
hexo: 3.8.0
hexo-cli: 1.1.0
os: Darwin 18.2.0 darwin x64
node: 11.7.0
```

### NexT Information

**NexT Version:**

- [ ] Latest Master branch
- [x] Latest Release version
- [ ] Old version

**NexT Scheme:**

- [ ] All schemes
- [ ] Muse
- [x] Mist
- [ ] Pisces
- [ ] Gemini

**Comment by stevenjoezhang** [comment]
Clicking the "sidebar-toggle" button is not the only way to close the sidebar. It can be done by swiping over the screen:
https://github.com/theme-next/hexo-theme-next/blob/7d7078c87fef1e5c3ac3ed138c4af45124be50d3/source/js/src/motion.js#L128-L134
Also, the feature to close sidebar when click blank area is already enabled on mobile platform:
https://github.com/iissnan/hexo-theme-next/commit/3a3b864f6616f89aaa0c1e92ed0b9f98bd96daf1
See:
https://github.com/theme-next/hexo-theme-next/blob/7d7078c87fef1e5c3ac3ed138c4af45124be50d3/source/js/src/motion.js#L90-L96
Maybe it's not necessary to add this in desktop platform

**Comment by stevenjoezhang** [comment]
Fixed in #607
